mlrun/db/httpdb.py

```diff
@@ -2814,12 +2813,30 @@ def store_marketplace_source(
         response = self.api_call(method="PUT", path=path, json=source)
         return schemas.IndexedMarketplaceSource(**response.json())
 
-    def list_marketplace_sources(self):
+    def list_marketplace_sources(
+        self,
+        item_name: Optional[str] = None,
+        tag: Optional[str] = None,
+        version: Optional[str] = None,
+    ):
         """
         List marketplace sources in the MLRun DB.
+
+        :param item_name:   Sources contain this item will be returned, If not provided all sources will be returned.
+        :param tag:         Must be provided with item name, will be filtered by this tag.
+        :param version:     Must be provided with item name, will be filtered by this version.
```

Comment:
I think this docstring is very confusing - it is read as "if you provide an item name you must also provide tag AND version", which IIUC is really not the case. This eventually goes to `get_source_catalog` which supports getting just one of those, or none of them as well. I believe what you meant was something like "Supported only if item name if provided"